src/ui/Alert/Alert.tsx

```diff
@@ -0,0 +1,104 @@
+import { cva, VariantProps } from 'cva'
+import * as React from 'react'
+
+import { cn } from 'shared/utils/cn'
+
+import successImage from './assets/check-circle.svg'
+import warningImage from './assets/exclamation.svg'
+import infoImage from './assets/information-circle.svg'
+import errorImage from './assets/x-circle.svg'
+
+const alertVariants = cva(
+  '[&>svg]:text-foreground relative w-full border-l-4 p-4 [&>svg+div]:translate-y-[-3px] [&>svg]:absolute [&>svg]:left-4 [&>svg]:top-4 [&>svg~*]:pl-7',
```

Comment:
I've never seen selectors used like this! Super cool, had no idea tailwind supported this.

That said, I'm wondering why these styles are here and not on the `img` tag or something else lower down?

Reply:
This is actually just some leftover cruft I forgot to remove when copying over the shadcn code!